examples/prometheus/README.md

```diff
@@ -116,3 +116,29 @@ Returns a running count (not a rate) of docker operations that have failed since
 > kubelet_pleg_relist_latency_microseconds
 
 Returns PLEG (pod lifecycle event generator) latency metrics.  This represents the latency experienced by calls from the kubelet to the container runtime (i.e. docker or CRI-O).  High PLEG latency is often related to disk I/O performance on the docker storage partition.
+
+### OpenShift build related queries
+
+> count(openshift_build_running_phase_start_time_seconds{} < time() - 360)
+
+Returns the number of builds that have been running for more than 10 minutes (360 seconds).
+
+> count(openshift_build_new_pending_phase_creation_time_seconds{} < time() - 360)
+
+Returns the number of build that have been waiting at least 10 minutes (360 seconds) to start.
+
+> count(openshift_build_failed_phase_total{})
+
+Returns the number of failed builds.
+
+> count(openshift_build_terminal_phase_total{phase="complete"})
+
+Returns the number of successfully completed builds.
+
+> openshift_build_failed_phase_total{}
+
+Returns the latest totals for failed builds.
```

Comment:
how is this different from the count() query above?

Reply:
if the count() query above is doing a sum of all the collected data points ("yesterday there were a total of 100 failed builds in the system", "today there were a total of 101", "count=201") that doesn't seem too useful since it's going to double (or triple or quadruple..) count a lot of builds.

Reply:
this query will return a multi-row table which includes count by reason; I'll push an update the clarifies that, as well as the aggregation that occurs with count (though that might need to be sum ... I'll sort that out as well).

Reply:
got it, thanks. (I think you explained that to me before in my cube, sorry i needed the refresher)

Reply:
np / yw